pwndbg/heap/structs.py

```diff
@@ -123,7 +131,7 @@ def __init__(
 
 class CStruct2GDB:
     code = gdb.TYPE_CODE_STRUCT
-    _c_struct: pwndbg.gdblib.ctypes.Structure = None
```

Comment:
I have not looked at this code in some time, but I think it was intentional to make this a class variable that gets overridden in child classes. I don't know what unintended effects there might be by changing that. cc: @CptGibbon in case you have thoughts on this.

Reply:
@lebr0nli I shall defer to you old chap

Reply:
I think my original type hint was wrong, thanks @peace-maker for fixing it here!
Since `_c_struct` is a `type` instead of an instance of `pwndbg.gdblib.ctypes.Structure`, also `ctypes.Structure` is the parent class of `pwndbg.gdblib.ctypes.Structure`, the way it's written in this commit makes more sense to me.

> I have not looked at this code in some time, but I think it was intentional to make this a class variable that gets overridden in child classes.

Yep, it's intentional.

> I don't know what unintended effects there might be by changing that.

Removing the `= None` here should not cause unintended effects I think. (At least all the tests related to this code passed :p)